### DISCUSSION TITLE
e2e test scenario for extension

### DISCUSSION THREAD

**Discussion by christian-bromann** [microsoft/vscode-jupyter#9575] [New Ideas]
Hey extension devs 👋 

I've been working on an [e2e test integration](https://github.com/webdriverio-community/wdio-vscode-service) into WebdriverIO, a test framework based on the W3C WebDriver standard. At my company we needed e2e testing for some of our extensions and I wonder if the you would be interested to add this to the Jupyter VSCode Extension as well. Tests can be run using GH Actions on Mac, Windows or Ubuntu. It seems that the project already has tons of unit tests and I am sure there are some e2e integration scenarios we could cover with e2e testing.

I am happy to take a stab at this if this would be something y'all would consider.

**Comment by DonJayamanne** [comment]
Closing in favor of https://github.com/microsoft/vscode-jupyter/issues/9584
